docs/search/index.mjs

```diff
@@ -2,14 +2,17 @@ import algoliasearch from 'algoliasearch';
 import fs from 'fs/promises';
 
 const indexJson = JSON.parse(await fs.readFile('./index.json'));
+const discussionsJson = JSON.parse(await fs.readFile('./discussions.json'));
 
 const key = process.env['ALGOLIA_KEY'];
 if ( ! key ) { throw Error(`No key!`); }
 const client = algoliasearch('M53HHHS0ZW', key);
 const index = client.initIndex('docs');
+const discussionsIndex = client.initIndex('discussions');
 
 const now = Date.now();
 const indexJson_lu = indexJson.map((x) => ({ ...x, lu: now }));
```

Comment:
Basically `indexJson` should be expanded to have the discussions query result

Reply:
https://github.com/reach-sh/reach-lang/pull/1113/commits/786581b9769b35184cccfaf16530fbcfb20b19dd#diff-492c64361dcbc9c0a47af787344fde877166ef1fecc66716d69710e201f666f9R63